README.md

```diff
@@ -1,59 +1,120 @@
+
+In general, any active theme on the site should contain the following folder structure:
 
 ```console
 .
-├── assets           # Main assets folder
-│   ├── img          # Images folder
-│   └── css          # CSS folder
-│        ├── build   # Compiled CSS folder
-│        └── src     # Single-purpose CSS files (cart, checkout, etc)
-│   ├── js           # JS folder
-│        ├── build   # Compiled JS folder
-│        └── src     # Single-purpose JS files
-│   ├── scss         # SCSS folder
-│        ├── core    # Core SCSS files
-│        ├── ...     # Other SCSS files grouped by purpose
-├── includes         # PHP files (classes, functions, etc)
-├── languages        # Translations folder
-├── style.css        # Theme CSS file, the compiled CSS should be located here
+├── assets           	# Main assets folder
+│   ├── img/**         	  # Image assets folder
+│   └── css            	  # Single-purpose CSS assets folder
+│        ├── build/**       		    
+│        └── src/**         		    
+│   ├── js             	  # JS assets folder
+│        ├── build/**       		    
+│        └── src/**         			
+│   ├── scss           	  # SCSS folder for the main stylesheet
+│        ├── core/**        
+│        ├── ...
+│        ├── style.scss 
+│        ├── style-editor.scss
+├── includes/**        	# PHP files (classes, functions, etc)
+├── languages/**       	# POT and translations folder
+├── functions.php    	# Theme functions file
+├── style.css        	# Theme CSS file compiled from SCSS
+├── style-editor.css	# Editor CSS file compiled from SCSS
 .
 ```
 
-## Scripts
+### Gutenberg blocks
+
+All custom-built blocks must be placed in a mu-plugin. The demo project contains a mu-plugin called `build-processes-demo-blocks` which exemplifies building two blocks called `build-processes-demo/foobar` and `build-processes-demo/spamham`, respectively.
+
+By separating the blocks from the theme into a mu-plugin, we can ensure that the blocks are always available on the site, even if the theme is changed (e.g., because of a future redesign or as part of the debugging process). Moreover, it forces us, as developers, to think about the blocks as a separate entity from the theme and to design the code as such thus making it easier to reuse them in other projects.
+
+The mu-plugin must contain enough CSS for a basic styling of the blocks, but it's fine if the theme provides its own CSS for maintaining the site's look-and-feel of the site. Take [Seedlet as an example](https://github.com/Automattic/themes/tree/trunk/seedlet/assets/sass/blocks) of the theme providing its own styling for the blocks.
```

Comment:
> The mu-plugin must contain enough CSS for a basic styling of the blocks,

Perhaps this is better worded as something like:

> The mu-plugin must contain enough CSS for the block to be functional and not appear broken. All other styling can be held in the mu-plugin or the theme.

That way we make it clear that we want the block to be working accross projects, but it doesn't have to look "good" out of the box on another project.